docs/src/prototypes/alerts/BannerAlerts.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+import {
+  Flex,
+  Provider,
+  RadioGroup,
+  ThemeAnimation,
+  RadioGroupItemProps,
+  Header,
+  Button,
+} from '@stardust-ui/react'
+
+import AnimatedBannerAlert from './AnimatedBannerAlert'
+import ComposeMessage from '../chatPane/composeMessage'
+
+type BannerName = 'info' | 'oof' | 'danger' | 'urgent' | 'critical'
+
+const bannerNames: BannerName[] = ['info', 'oof', 'danger', 'urgent', 'critical']
+
+const bannerRadioItems: RadioGroupItemProps[] = bannerNames.map(bannerName => ({
+  key: bannerName,
+  value: bannerName,
+  label: `${_.startCase(bannerName)} banner`,
+}))
+
+const isAlertClosable = (bannerName: BannerName) =>
+  bannerName !== 'critical' && bannerName !== 'urgent'
+
+interface BannerAlertsState {
+  selectedBanner: RadioGroupItemProps
+}
+
+class BannerAlerts extends React.Component<{}, BannerAlertsState> {
+  private readonly initialState: BannerAlertsState = { selectedBanner: bannerRadioItems[0] }
+
+  state = this.initialState
+
+  render() {
+    const { selectedBanner } = this.state
```

Comment:
Can we store `bannerName` in state instead of props? Currently, it shows a bad practice because the state will contain event handlers and children:
![image](https://user-images.githubusercontent.com/14183168/54745108-7124dc80-4bd1-11e9-8ae4-78f1abcf140a.png)

Reply:
ooops, fixed, thanks!